eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -247,11 +247,24 @@ class Setup(datadir: File,
           feeratesRetrieved.trySuccess(Done)
       })
       _ <- feeratesRetrieved.future
+      (zmqBlocks_opt, zmqTxs_opt) <- bitcoin match {
+        case Bitcoind(client) => client.invoke("getzmqnotifications").map {
```

Comment:
This should be done before, along with all other bitcoind-related validations. See:

https://github.com/ACINQ/eclair/blob/use_getzmqnotification-pm/eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala